### PR TITLE
Add clickable cards block

### LINKED ISSUE #21: Clickable cards
Screenshot 2024-06-07 at 10 40 40

### Acceptance criteria:
- Card is clickable
- Button (with the same href can be added)
- Allow for multiple cards
- Switch responsively from row to column layout below a specified breakpoint
- Use default-content styling for the content

### PR DESCRIPTION
**Pull request by kronnox**
Resolve #21

Test URLs:
- Before: https://main--shredit--stericycle.aem.page/
- After: https://issue-21--shredit--stericycle.aem.page/
- After: https://issue-21--stericycle-shared--aemsites.hlx.page/en-us

### CODE REVIEW

File: blocks/clickable-columns/clickable-columns.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+.clickable-columns > div > .columns-img-col {
+  order: 0;
+}
+
+.clickable-columns > div > .columns-img-col img {
+  display: block;
+}
+
+@media (width >= 900px) {
```
Comment (ericvangeem) [inline blocks/clickable-columns/clickable-columns.css:31]: @kronnox there may be a browser compat issue with this style of media query as we found on a different project. Please see @seanathero's slack comment for more detail: https://herodigital.slack.com/archives/C06JE9U7Y74/p1717612144522739

**Review by ericvangeem — APPROVED** [review]

**Comment by seanathero** [comment]
@dkuntze @kronnox Note the current site provides an aria-label attr on the links to provide more context to screen readers. I think we lose that here.

**Comment by dkuntze** [comment]
> @dkuntze @kronnox Note the current site provides an aria-label attr on the links to provide more context to screen readers. I think we lose that here.

We can fix that

**Comment by kronnox** [comment]
> Note the current site provides an aria-label attr on the links to provide more context to screen readers. I think we lose that here.

@seanathero Taking care of that in a separate PR (if code changes are even necessary) to not further block merging here.

**Pull request merged**